### PR TITLE
TACT-162: Page twitches when calling an additional menu

### PR DESCRIPTION
**Pull request by al-petrushin**
**Describe the issue**

[TACT-162](https://linear.app/octolab/issue/TACT-162/page-twitches-when-calling-an-additional-menu)

**Describe the pull request**

Added a parent container for the menu portal. 
Looks like a hack and is a hack. 🧐
But it works and now the menu increases the parent container and accordingly the scrolling.

### CODE REVIEW

**Review by kamilsk — CHANGES_REQUESTED** [review]
I blocked it, need to think

File: components/shared/TasksList/components/TaskItemMenu/index.tsx [hunk 1]
```diff
@@ -345,13 +348,16 @@ export const TaskItemMenu = observer(function TaskItemMenu() {
     store.openMenu();
   }, [onOpen, store]);
 
+  console.log('console.log(menuPortalRef)', menuPortalRef)
```
Comment (malakhovmisha) [inline components/shared/TasksList/components/TaskItemMenu/index.tsx:351]: no console

**Comment by kamilsk** [comment]
fixed in next pr #328

**Comment by kamilsk** [comment]
thanks for contribution!

**Pull request closed without merging**